CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to StackStorm pack directory
+
+You've decided to create or update a StackStorm integration pack. That's great! 
```

Comment:
As mentioned on Slack, I would prefer to move this to the documentation in the existing development docs section. Potentially this also requires some updates to the existing docs because we already have some docs on pack development. Then we can just link to it from this file.

This way most of the docs are located in a single place (less duplication) and harder for things to get out of sync.